src/sentry/models/organization.py

```diff
@@ -159,6 +159,10 @@ class Organization(Model, SnowflakeIdMixin):
                 "early_adopter",
                 "Enable early adopter status, gaining access to features prior to public release.",
             ),
+            (
+                "codecov_access",
+                "Enable codecov integration.",
+            ),
```

Comment:
You need to add this to the end, otherwise you'll shift all existing bits forward.

You should also verify that the bit you end up using is 0 for all orgs in the database.

Reply:
modified and verified, thanks!